pkgs/applications/audio/ptcollab/default.nix

```diff
@@ -0,0 +1,34 @@
+{ mkDerivation
+, stdenv
+, fetchFromGitHub
+, qmake
+, qtbase
+, qtmultimedia
+, libvorbis
+}:
+
+mkDerivation rec {
```

Comment:
`rec` is unneeded now, but I guess you put it because it will be when switching to a release